mne/io/snirf/tests/test_snirf.py

```diff
@@ -503,3 +503,37 @@ def test_birthday(tmp_path, monkeypatch):
 
     raw = read_raw_snirf(fname)
     assert raw.info["subject_info"]["birthday"] == (1950, 1, 1)
+
+
+@requires_testing_data
+def test_sample_rate_jitter(tmp_path):
+    """Test handling of jittered sample times."""
+    from shutil import copy2
+
+    # Create a clean copy and ensure it loads without error
+    new_file = tmp_path / "snirf_nirsport2_2019.snirf"
+    copy2(snirf_nirsport2_20219, new_file)
+    read_raw_snirf(new_file)
+
+    # Edit the file and add jitter within tolerance (0.99%)
+    with h5py.File(new_file, "r+") as f:
+        orig_time = np.array(f.get("nirs/data1/time"))
+        acceptable_time_jitter = orig_time.copy()
+        average_time_diff = np.mean(np.diff(orig_time))
+        acceptable_time_jitter[-1] += 0.0099 * average_time_diff
+        del f["nirs/data1/time"]
+        f.flush()
+        f.create_dataset("nirs/data1/time", data=acceptable_time_jitter)
+    read_raw_snirf(new_file)
```

Comment:
```suggestion
    with use_log_level("info"), catch_logging() as log:
        read_raw_snirf(new_file)
    assert "Found jitter of 0.990000%" in log.getvalue()
```

(with associated `from mne.utils import catch_logging, use_log_level` at the top)

This won't fix the CI failure but it makes sure our jitter detection is actually working

Reply:
Feel free to push I'm still messing with conda envs